vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -259,7 +259,9 @@ void Publisher::SaveVisitInternal(
         publisher_info->favicon_url = fav_icon;
     }
   } else {
-    publisher_info->favicon_url = ledger::kClearFavicon;
+    if (publisher_info->favicon_url.empty()) {
```

Comment:
This should check for !is_verified and always clear if that is the case (discussed with @NejcZdovc)